### PR TITLE
Added new option from travis-ci/dpl#677

### PR DESCRIPTION
**Pull request by MathiasSM**
Added new `deployment-file` optional boolean option from travis-ci/dpl#677

### CODE REVIEW

File: user/deployment/pages.md [hunk 1]
```diff
@@ -44,19 +44,21 @@ or via [encrypted variables in
...
+* `name`: Optional, committer, defaults to `Deployment Bot`.
 * `committer-from-gh`: Optional, defaults to `false`. Allows to use token's
   owner name and email for commit. Overrides `email` and `name` options.
 * `allow-empty-commit`: Optional, defaults to `false`. Enabled if only
   `keep-history` is `true`.
-* `github-url`: Optional, the URL of the self-hosted GitHub enterprise, defaults to `github.com`
+* `github-url`: Optional, the URL of the self-hosted GitHub enterprise, defaults to `github.com`.
 * `verbose`: Optional, be verbose about internal steps, defaults to `false`.
+* `deployment-file`: Optional, defaults to `false`, enables creation of deployment-info files.
+* `github-url`: Optional, the URL of the self-hosted GitHub enterprise, defaults to `github.com`.
```
Comment (webknjaz) [inline user/deployment/pages.md]: It's a duplicate

Comment (MathiasSM) [reply user/deployment/pages.md]: 🤦‍♂️

Comment (MathiasSM) [reply user/deployment/pages.md]: Already squashed the error (if that's a thing)

**Comment by plaindocs** [comment]
@BanzaiMan I guess I can merge this now?

**Comment by webknjaz** [comment]
The feature is in production, so it's better to update the docs ASAP

**Comment by plaindocs** [comment]
Hey @webknjaz looks like I lost this PR somehow. 

Would you mind fixing the merge conflict and I'll merge it in.

**Comment by webknjaz** [comment]
@plaindocs it's not mine. It's @MathiasSM's.

**Comment by webknjaz** [comment]
@plaindocs ^

**Comment by plaindocs** [comment]
Apologies for the mixup folks. Thanks for the PR and comments.

**Pull request merged**